.env

```diff
@@ -1,16 +1,25 @@
 # This file is commited. Do not store secrets here
 
-REACT_APP_ENV=staging
-REACT_APP_QUERY_NODE_URL=https://sumer-dev-2.joystream.app/query/server/graphql
-REACT_APP_QUERY_NODE_SUBSCRIPTION_URL=wss://sumer-dev-2.joystream.app/query/server/graphql
-REACT_APP_ORION_URL=https://orion-staging.joystream.app/graphql
-REACT_APP_NODE_URL=wss://sumer-dev-2.joystream.app/rpc
-REACT_APP_FAUCET_URL=https://sumer-dev-2.joystream.app/members/register
+REACT_APP_ENV=development
 
-#REACT_APP_ENV=production
-#REACT_APP_QUERY_NODE_URL=https://hydra.joystream.org/graphql
-#REACT_APP_QUERY_NODE_SUBSCRIPTION_URL=wss://hydra.joystream.org/graphql
-#REACT_APP_ORION_URL=https://orion.joystream.org/graphql
-#REACT_APP_NODE_URL=wss://rome-rpc-endpoint.joystream.org:9944/
-#REACT_APP_FAUCET_URL=https://member-faucet.joystream.org/register
+# default is staging
+REACT_APP_STAGING_QUERY_NODE_URL=https://sumer-dev-2.joystream.app/query/server/graphql
+REACT_APP_STAGING_QUERY_NODE_SUBSCRIPTION_URL=wss://sumer-dev-2.joystream.app/query/server/graphql
+REACT_APP_STAGING_ORION_URL=https://orion-staging.joystream.app/graphql
+REACT_APP_STAGING_NODE_URL=wss://sumer-dev-2.joystream.app/rpc
+REACT_APP_STAGING_FAUCET_URL=https://sumer-dev-2.joystream.app/members/register
+
+# these two url are only temporary. We don't want to break the app when switching to 'mocking'
```

Comment:
I don't understand this comment, why do we need valid values for query node urls?